internal/cli/atlas/backup/restores/restores.go

```diff
@@ -29,6 +29,7 @@ func Builder() *cobra.Command {
 
 	cmd.AddCommand(
 		ListBuilder(),
+		DescribeBuilder(),
```

Comment:
[q] Since this is a new command under atlas, shouldn’t we add it only in atlascli?

Reply:
done 👍